css/style.css

```diff
@@ -151,68 +150,67 @@ button {
 .storeContainer img {
   width: 50%;
 }
-.leftForm {
-  width: 50%;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-}
 .storeTitle {
   font-weight: 400;
   color: var(--primary-color);
 }
+.leftForm {
+  width: 50%;
+}
 
 form {
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: 80%;
+  width: 75%;
 }
 
 label {
   font-size: 1rem;
   color: #434242;
   margin-bottom: 0.5rem;
+  font-weight: 400;
 }
```

Comment:
it would be nice to make a class for your form, and specify your label within the form class. So you wouldn't be confused as this can affect other part of your site!